src/mlpack/core/metrics/bleu_impl.hpp

```diff
@@ -187,11 +187,9 @@ ElemType BLEU<ElemType, PrecisionType>::Evaluate(
 
 template <typename ElemType, typename PrecisionType>
 template <typename Archive>
-void BLEU<ElemType, PrecisionType>::serialize(
-    Archive& ar,
-    const unsigned int /* version */)
+void BLEU<ElemType, PrecisionType>::serialize(Archive& ar, std::uint32_t const version)
```

Comment:
I think this line is longer than 80 characters; also, `version` is unused so we can comment that out. :+1:

Reply:
Yes, of course